assets/js/common/Tooltip/Tooltip.jsx

```diff
@@ -26,6 +26,7 @@ function Tooltip({
   place = 'top',
   isEnabled = true,
   wrap = true,
+  visible,
```

Comment:
I am wondering whether this prop is somehow doing something similar to what `isEnabled` does.

Maybe we can consolidate in only one of the two. What do you think?

Reply:
**Short answer:** 

`visible` can force the element to show or not while `isEnabled` controls whether is in the virtual dom or not.

**Long answer:**

There is some hidden knowledge here I had to find out. The `isEnabled` flag just don't render the tooltip into the virtual dom. The `visible` flag renders the components in the virtual dom, but hides it with css.

The reason we need both is that using `isEnable` would re-render children and make the input field to lose focus.
Furthermore, `visible` is a flag that forces visibility regardless of the user action (that is, the popup is shown even when the user doesn't hover the control).

The `visible` flag is a prop from our dependency `rc-trigger` ([link](https://github.com/fis-components/rc-trigger/blob/7b659fd7f76ad88129ca09e56522b3419fcf66ea/lib/Popup.js#L78-L79)).

Reply:
Btw you made me think it might be worth a comment in https://github.com/trento-project/web/pull/2696/commits/a72d5844bdf57261dbc2af988a3fe115122a9321.

Thanks

Reply:
Ok, if we need both, so be it. Thanks.